index.js

```diff
@@ -0,0 +1,51 @@
+const isPlainObj = require("is-plain-obj");
+const TARGET_BROWSER = "browser";
+
+module.exports = function preset(context, opts = {}) {
+  // see https://github.com/facebook/create-react-app/blob/590df7eead1a2526828aa36ceff41397e82bd4dd/packages/babel-preset-react-app/index.js#L52
+  const env = process.env.BABEL_ENV || process.env.NODE_ENV;
+  if (env !== "development" && env !== "test" && env !== "production") {
+    throw new Error(
+      "Using `@mapbox/babel-preset-mapbox` requires that you specify `NODE_ENV` or " +
+        '`BABEL_ENV` environment variables. Valid values are "development", ' +
+        '"test", and "production". Instead, received: ' +
+        JSON.stringify(env) +
+        "."
+    );
+  }
+
+  opts = isPlainObj(opts) ? opts : {};
+
+  var isEnvDevelopment = env === "development";
+  var isEnvProduction = env === "production";
+  var isEnvTest = env === "test";
+
+  const target = opts.target || TARGET_BROWSER;
+
+  const presets = [
+    [require.resolve("babel-preset-env"), opts.env],
```

Comment:
We should definitely have some defaults for this. Hopefully most users will not have to research `babel-preset-env` at all.

Reply:
One important thing to keep in mind here is that for Webpack & Rollup you'll actually want to set `modules: false` so Babel doesn't compile ES2015 modules.

Reply:
Keep `modules: true` for the test environment. But then the Browserify user also needs `modules: true`. So it's kind of tricky.

Reply:
To keep the code base simple, how about we ask Browserify users to override the `babel-preset-env` manually in their `.babelrc` ?

Reply:
Yeah, sure.